Cargo.toml

```diff
@@ -12,37 +12,21 @@ readme = "README.md"
 edition = "2018"
 
 [features]
-default = ["cursor", "style","terminal","screen","input"]
+default = ["cursor", "style", "terminal", "screen", "input"]
 
 cursor = ["crossterm_cursor"]
 style = ["crossterm_style"]
 terminal = ["crossterm_terminal"]
 screen = ["crossterm_screen"]
 input = ["crossterm_input"]
 
-[workspace]
-
-members = [
-    "crossterm_winapi",
-    "crossterm_utils",
-    "crossterm_cursor",
-    "crossterm_style",
-    "crossterm_terminal",
-    "crossterm_input",
-    "crossterm_screen"
-]
-
-exclude = [
-    "examples/program_examples"
-]
-
 [dependencies]
-crossterm_screen =   { path = "./crossterm_screen", version = "0.3.0"  , optional = true  }
```

Comment:
I have the following file structure for crossterm:

```
- crossterm-rs
   - crossterm
   - crossterm_cursor
   - etc..
```

Having a path as a reference is easier for development it allows you to change things on the fly. I would prefer to keep `path = "../crossterm_cursor"`, so that the reference to this crate is in the folder `crossterm-rs` and point to `crossterm_cursor`. If we don't add it back, you need to add the path here each time you change something in a subcrate and remove it when creating a PR.